05week/spaceTravelToMars.js

```diff
@@ -10,6 +10,38 @@ let jobTypes = {
 };
 
 // Your code here
+class CrewMember {
+  constructor(name, job, specialSkill) {
+    this.name = name
+    this.job = job
+    this.specialSkill = specialSkill
+    this.ship = null
+  }
+
+  enterShip(someShip){
+    this.ship = someShip
+    someShip.crew.push(this)
+    console.log(someShip, this.ship.crew[0].ship)
+  }
+}
+
+class Ship {
+  constructor(name, type, ability) {
+    this.name = name
+    this.type = type
+    this.ability = ability
+    this.crew = []
+  }
+
+  missionStatement() {
+      //this method should return the ship's ability if there is a crew memeber whose job matches up with the ships type
```

Comment:
You need to check that there is a crew member whose job matches the ship's ability.